### DISCUSSION TITLE
Two questions and an issue

### DISCUSSION THREAD

**Discussion by magnificar** [TT-ReBORN/Georgia-ReBORN#163] [Q&A]
Hello! Recently I've installed this theme, and it's great, I love it! It's a big improvement over the original Georgia, imo.
I've been learning how to customize it how I like it and it's been fun. However there are two things I can't figure out.

1. Is there a way to show the %EDITION% tag in the playlist album header, like the old Georgia does? It did it like this:
![2023-12-27 (1)](https://github.com/TT-ReBORN/Georgia-ReBORN/assets/61425748/afa55f77-4336-4ac9-9394-a6c35bdb4f03)
In Reborn, I've found out that if you go right-click > Playlist options menu > Album header and uncheck Show disc sub-header, the tag appears: 
![2024-01-07 (1)](https://github.com/TT-ReBORN/Georgia-ReBORN/assets/61425748/3b1340b5-3e05-4ebb-bea2-2998feda35f9)
But when you check that option again, it vanishes:
![2024-01-07 (3)](https://github.com/TT-ReBORN/Georgia-ReBORN/assets/61425748/512c6688-5d8f-41c8-b3b6-f446793cee86)
I personally like the disc sub-header, and would prefer to show the %EDITION% tag alongside it. Is there a way it could be done?

2. Is there a way to use custom search fields for the biography module, like you can with the lyrics? 

Now, for the issue.
When you're in Details mode, and you click the Lyrics button on the top of the window, you get thrown into the playlist mode. But you can still use lyrics by going right-click > Show/hide lyrics. Like this:
![2024-01-07 (6)](https://github.com/TT-ReBORN/Georgia-ReBORN/assets/61425748/7fe2a7af-0582-40c0-b4f3-c47616fefdfb)
I'm not sure if that's a bug with the button or if you're not supposed to use the Lyrics panel while in Details mode. I hope the latter isn't the case because I quite like this look! 

That's it for now, thanks for the help and for the great work!

**Comment by d3rb** [comment]
I found the problem that @relampo describes. If "Remember active lyrics state" is enabled, this error occurs. But I also have to admit that I like the view of details with lyrics, even if that's certainly not the intention.

Here is a short video showing the problem:

https://mega.nz/file/a8khBZRC#BHiWwnaArTzo-D1htFRgXFwI2ARtHY-E1RTSUORAmYY

**Reply by magnificar** [reply]
Hi, thanks for your answer - I somehow missed the "Remember active lyrics state" option! It makes the theme work closer to what I wanted. I managed to reproduce the bug you showed, it happens only when you have the Library layout set to full or split. It does not happen when it's set to normal.

**Comment by TT-ReBORN** [comment]
Hi @relampo,

glad you like the theme. There was a change in the playlist header album subtitle behavior in the past, 
but I have recently restored it and it's now available in the master version.

Displaying lyrics in the Biography panel is not the same as the Lyrics panel. Bio lyrics are fetched from the Lyrics panel, that means it 
is just a text reader. The Lyrics panel uses its own user-component 'ESLyric' that has its custom search input field and other lyrics options.

You can display the lyrics from the Lyrics panel ( top menu > Lyrics ) and also from the Details panel ( top menu > Details ).
The Lyrics panel displays always the album art with the lyrics overlay and the Playlist, or just the album art with lyrics if you right click and change to full width. The Details panel displays the metadata grid with album art and disc, but it can also display lyrics in this panel only, 
if you right click on the album art > Display lyrics. If you are in the Playlist panel ( no active top menu buttons ) and right click on the album art there, it will switch and display the Lyrics panel.

Hope that helps.

P.S I would wait a little longer before updating to the latest master version, need to finish and fix some remaining stuff.
I will let you guys know in the [DEV TESTING thread here](https://github.com/TT-ReBORN/Georgia-ReBORN/discussions/74 ).

-TT

**Reply by magnificar** [reply]
Hi TT, thanks for the answer and for your work on the theme. I will wait for the next version.

I did not explain myself properly regarding the biography. I apologize for that. My use case is as follows: I use MusicBrainz Picard to tag my files. When it gets onto a track with multiple artists, it will tag both of them on the %ARTIST% tag. Resulting in something like "Frank Sinatra feat. Kendrick Lamar" (or _featuring_, or _with_, or _and_, MusicBrainz doesn't have a standard regarding that, and that's ignoring other languages). The biography panel will then attempt to search this tag, finding (predictably) no photos and very little info. A similar thing also happens (and breaks) the lyrics search function, but I managed to find a workaround. Picard also uses the %ARTISTS% tag to write the same artist list but itemized. Like this: "Frank Sinatra; Kendrick Lamar". So, since ESLyric has the option custom search fields (in Preferences > Tools > ESLyric > Lyric Option > Search Settings), I just use $meta(artists,0) to get the main artist and then the lyrics search work flawlessly. I was wondering if something like this is also possible to the biography search, if I could customize the search field for it. I hope I was clear this time, and sorry for any confusions, English is not my first language. Maybe in the future, this could be used to get multiple artist to the biography panel, but that's not a priority I think.

Regarding the lyrics panel, I understand now how it's supposed to work. In my opinion, it would be better if the lyrics button at the top menu made the lyrics appear without disabling the details panel. But it's not a big deal, to be honest.

Thanks for the support, take care.

**Comment by TT-ReBORN** [comment]
I don't know what program you use to tag your files, but there is no issue if you tag your music properly.

---

### The correct way would be to use `%original artist%` tag:

![tagging1](https://github.com/TT-ReBORN/Georgia-ReBORN/assets/79368795/e88c981e-28ed-4f05-ab2f-430e151cd9c2)

![tagging1_playlist](https://github.com/TT-ReBORN/Georgia-ReBORN/assets/79368795/7ddd8909-92ee-46b5-8c0e-a9791e9b52a9)

---

### If one album has multiple artists, you use `%album artist%` for the main artist and `%artist%` for the individual track artist:
![tagging2](https://github.com/TT-ReBORN/Georgia-ReBORN/assets/79368795/1b9a79fb-f8d2-4074-ac3b-4df672c98751)

![tagging2_playlist](https://github.com/TT-ReBORN/Georgia-ReBORN/assets/79368795/ce8e1ac5-c3e6-417c-a771-a6bf61e89b85)

### For this setup, activate also top menu `Options` > `Playlist` > `Track row` > `Show artist name on difference`

---

### Multi-values work also just fine for `%artist%` and `%album artist%` if you split them.

---

The Details panel and Lyrics panel are two separate panels. The Lyrics panel has different options then the Details panel and also
feature a full width layout. It is just an extra feature in the Details panel to display also the lyrics there. It is much better as it is now
with the Playlist for quick navigation, as opposed when switching directly to Details with lyrics.

I hope that helps.

-TT

**Comment by TT-ReBORN** [comment]
P.S: You can download the current master version:
https://github.com/TT-ReBORN/Georgia-ReBORN/archive/refs/heads/master.zip
It now has the original playlist header album subtitle behavior as it was with some bug fixes.
